framework/test/unit/modules/pos/endpoint.spec.ts

```diff
@@ -175,6 +175,34 @@ describe('PosModuleEndpoint', () => {
 				expect(stakerDataReturned.pendingUnlocks[0].amount).toBeString();
 			});
 		});
+
+		describe('when staker address is invalid', () => {
+			it.each([
+				{
+					input: "lisk1'0",
+					expected: 'Address length does not match requirements. Expected 41 characters.',
+				},
+				{
+					input: 'invalidAddress',
+					expected: 'Address length does not match requirements. Expected 41 characters.',
+				},
+				{
+					input: '',
+					expected: 'Address length does not match requirements. Expected 41 characters.',
+				},
```

Comment:
This identical array of objects is used in 2 test cases. Furthermore, the `expected` property always contains the same string.

I think the test would look cleaner if you remove duplicated data, make only 1 array with invalid addresses, and have one constant with the error message, e.g.

```typescript
const invalidAddresses = ["lisk1'0", "invalidAddress", ""]
const ADDRESS_LENGTH_ERROR = 'Address length does not match requirements. Expected 41 characters.'
```

Then have both test cases use these 2 constants.

Reply:
They are the same if you ask me. test.each is a way of running parameterized tests just like this one. I think this is a preference issue. It's a cleaner way as well to run parameterized tests 😎

Reply:
I didn't say thay you shouldn't use `it.each()` 🙂

What I meant was that you could simplify the data structure you're passing to it and remove duplication.

IMO, it will improve the maintainability of the test suite.